package.json

```diff
@@ -39,7 +39,7 @@
     "karma-nodeunit": "~0.2",
     "karma-requirejs": "~0.2",
     "karma-story-reporter": "~0.3",
-    "kexec": "^2.0.2",
+    "kexec": "git@github.com:ably-forks/node-kexec.git#update-for-node-12",
```

Comment:
AFAIUI we're not forking this because we want to make changes - why don't we just point to someone else's fork?

Reply:
I kinda thought we had a policy of cloning to ably-forks whenever we want to reference someone's github branch as a client lib dependency, in case the original repo is deleted. (at least for things where the referenced repo is not unlikely to be deleted, ie if it isn't the original project repo etc)

Reply:
@QuintinWillison I'd welcome your input on this.

TBH if the fork is owned by what appears to be a stable organisation then I don't think we need to create a fork for ourselves. Perhaps though we do need to decide case-by-case whether or not that's true. It also depends on how long we think we're likely to need to use a fork.

Reply:
I would err on the side of defensive and never assume any third party is stable.